browser_patches/checkout_build_archive_upload.sh

```diff
@@ -397,18 +413,26 @@ BUILD_ALIAS="$BUILD_FLAVOR r$BUILD_NUMBER"
 send_telegram_message "$BUILD_ALIAS -- started"
 
 if generate_and_upload_browser_build 2>&1 | ./sanitize_and_compress_log.js $LOG_PATH; then
-  # Report successful build. Note: we don't know how to get zip size on MINGW.
+  # Report successful build. Note: MINGW might not have `du` command.
   UPLOAD_SIZE=""
   if command -v du >/dev/null && command -v awk >/dev/null; then
     UPLOAD_SIZE="$(du -h "$ZIP_PATH" | awk '{print $1}') "
   fi
   send_telegram_message "$BUILD_ALIAS -- ${UPLOAD_SIZE}uploaded"
 
   # Check if we uploaded the last build.
-  if ./check_cdn.sh $BROWSER_NAME > /dev/null; then
-    LAST_COMMIT_MESSAGE=$(git log --format=%s -n 1 HEAD -- ./$BROWSER_NAME/BUILD_NUMBER)
-    send_telegram_message "<b>$BROWSER_NAME r${BUILD_NUMBER} COMPLETE! ✅</b> $LAST_COMMIT_MESSAGE"
-  fi
+  (
+    for i in $(cat "${BROWSER_NAME}/${BUILDS_LIST}"); do
+      URL="${HOST}/${BROWSER_NAME}/${BUILD_NUMBER}/$i"
+      if ! [[ $(curl -s -L -I $URL | head -1 | cut -f2 -d' ') == 200 ]]; then
+        # Exit subshell
+        echo "Missing build at ${URL}"
+        exit
+      fi
+    done;
+    LAST_COMMIT_MESSAGE=$(git log --format=%s -n 1 HEAD -- "./${BROWSER_NAME}/BUILD_NUMBER")
```

Comment:
Unrelated to this PR but this message oftentimes is unrelated to the last browser patch, maybe we can fix that too?

Reply:
> Unrelated to this PR but this message oftentimes is unrelated to the last browser patch, maybe we can fix that too?

Somehow git on mingw ignores arguments after the `--` - I'll look into that separately (requires booting win)